docs/source/tutorial_mor_with_anns.md

```diff
@@ -404,8 +403,131 @@ coefficients. In the same fashion, there exists a
+Squares represent layers similar to those used in feedforward neural networks, where inside the
+square the applied activation function is mentioned, whereas circles denote element-wise
+operations like element-wise multiplication ({math}`\times`), element-wise addition ({math}`+`) or
+element-wise application of the hyperbolic tangent function ({math}`\tanh`). The filled black
+circle represents the concatenation of the inputs. Furthermore, {math}`\sigma` is the sigmoid
+activation function ({math}`\sigma(x)=\frac{1}{1+\exp(-x)}`), and {math}`\tanh` is the hyperbolic
+tangent activation function ({math}`\tanh(x)=\frac{\exp(x)-\exp(-x)}{\exp(x)+\exp(-x)}`) used for
+the respective layers in the LSTM network. Finally, the layer {math}`P` denotes a projection layer
+that projects vectors of the internal size to the hidden respectively output size. Hence,
+internally, the LSTM can deal with larger quantities and finally project them onto a space with a
+desired size.
+
+We will take a closer look at the individual components of an LSTM cell in the subsequent
+paragraphs.
+
+##### The forget gate
+```{image} lstm_cell_forget_gate.svg
+:alt: Forget gate of an LSTM cell
+:align: right
+```
+As the name already suggests, the *forget gate* determines which part of the cell state
+{math}`c_{k-1}` the network forgets when moving to the next cell state {math}`c_k`. The main
+component of the forget gate is a neural network layer consisting of an affine-linear function
+with adjustable weights and biases followed by a sigmoid nonlinearity. By applying the sigmoid
+activation function, the output of the layer is scaled to lie between 0 and 1. The cell state
+{math}`c_{k-1}` from the previous cell is (point-wise) multiplied by the output of the layer in
+the forget gate. Hence, small values in the output of the layer correspond to parts of the cell
+state that are diminished, while values near 1 mean that the corresponding parts of the cell
+state remain intact. As input to the forget gate serves the pair {math}`(h_{k-1},\mu(t_k))`.
+
+##### The input gate
+```{image} lstm_cell_input_gate.svg
+:alt: Input gate of an LSTM cell
+:align: right
+```
+To further change the cell state, an LSTM cell contains a so called *input gate*. This gate mainly
+consists of two layers, a sigmoid layer and an hyperbolic tangent layer. As in the forget gate,
+the sigmoid layer determines which parts of the cell state to adjust. On the other hand, the
+hyperbolic tangent layer determines how to adjust the cell state. Using the hyperbolic tangent as
+activation function scales the output to be between -1 and 1, and allows for small updates of the
+cell state. To finally compute the update, the outputs of the sigmoid and the hyperbolic tangent
+layer are multiplied entry-wise. Afterwards, the update is added to the cell state (after the cell
+state passed the forget gate). The new cell state is now prepared to be passed to the subsequent
+LSTM cell.
```

Comment:
Maybe also describe the inputs of this gate, as done above? Also maybe further above mention the outer input and output of the LSTM in a tuple like way?

Reply:
Would you mind taking another look at this part @TiKeil (once the new version of the documentation is built)? At the end of the general paragraph about the LSTM cell, I added a short note on the input and the output of an individual LSTM cell.